src/utils/truex_servers.js

```diff
@@ -0,0 +1,58 @@
+
+export function isTruexProductionUrl(url) {
+    if (url) {
+        const m = url.match(/^(https?:\/\/)?([a-z])+.truex.com/);
```

Comment:
nit: I think at some point management was thinking about changing our name... Wish this wasn't hardcoded. 
But don't think this actually matters? As... changing the domain name probably won't be an easy task to begin with anyway.